### PR TITLE
modify 'terasoluna-gfw-parent-pom.xml' for Maven Central repository. #425

### PR DESCRIPTION
**Pull request by btmiyashitasts**
Please review #425

### CODE REVIEW

File: terasoluna-gfw-parent/pom.xml [hunk 1]
```diff
@@ -21,6 +21,18 @@
         <name>terasoluna.org</name>
         <url>http://terasoluna.org</url>
     </organization>
+    <developers>
+        <developer>
+            <name>NTT DATA</name>
+            <organization>NTT DATA, Inc.</organization>
```
Comment (ikeyat) [inline terasoluna-gfw-parent/pom.xml]: - [ ] Please change to "NTT DATA"

**Comment by btmiyashitasts** [comment]
Please review #425

**Comment by kazuki43zoo** [comment]
@ikeyat 

Check please !!

**Comment by ikeyat** [comment]
the changes look OK.
Please resolve conflict

**Comment by kazuki43zoo** [comment]
@ikeyat thanks.

@btmiyashitasts 
Please resolve conflict.

**Comment by ikeyat** [comment]
Stay this PR after @btyoshikawaa completes #425.

**Comment by btyoshikawaa** [comment]
(In reference)

**Comment by ikeyat** [comment]
Done within #606

**Pull request closed without merging**